scripts/infra_k8s_check.sh

```diff
@@ -461,5 +461,5 @@ function main(){
 
 main
 if ! ${ALADDIN_DEV:-false} ; then
-    "$(cd "$(dirname $0)" ; pwd)"/install-aladdin.sh
+    env -u PYTHONPATH "$(cd "$(dirname $0)" ; pwd)"/install-aladdin.sh
```

Comment:
we want to make sure `PYTHONPATH` is not set for the install script so we don't confuse `pipx`